### DISCUSSION TITLE
Inconsistent defaults in decimation algorithms between PyVista and VTK

### DISCUSSION THREAD

**Discussion by paulapatience** [pyvista/pyvista#4002] [Q&A]
The [normals attribute](https://vtk.org/doc/nightly/html/classvtkQuadricDecimation.html#a8c0eff59df4dd18e6387c0063ac74ebc) in `vtkQuadricDecimation` is enabled by default (in fact, all the attributes are enabled by default, though the overarching attribute error metric is disabled), but in PyVista's [`PolyDataFilters.decimate`](https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyDataFilters.decimate.html) it isn't (whereas the other attributes are). Is this intended?

Also, [`vtkDecimatePro::New`](https://vtk.org/doc/nightly/html/classvtkDecimatePro.html#a3e0e5e61782d90cd29e1103b24d085e2) sets the default feature angle to 15, but Pyvista's [`PolyDataFilters.decimate_pro`](https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyDataFilters.decimate_pro.html#decimate-pro) sets it to 45. Is this intended, perhaps to be consistent with [`PolyDataFilters.smooth`](https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyDataFilters.smooth.html#smooth) and [`PolyDataFilters.smooth_taubin`](https://docs.pyvista.org/api/core/_autosummary/pyvista.PolyDataFilters.smooth_taubin.html)'s defaults of 45?

**Comment by banesullivan** [comment]
I'm not too sure about the reasoning behind the decimate defaults differences

As for the feature angle choice of 45, I think this was chosen somewhat arbitrarily when the algorithms were implemented in https://github.com/pyvista/pyvista/pull/194 and 45 was likely chosen for consistency and being an intuitive cutoff for sharp angles (half of 90)... just my initial thoughts
